### PR TITLE
[Shop] Add ability to show static content

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | -
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ContentBundle/Doctrine/ODM/PHPCR/StaticContentRepository.php [hunk 1]
```diff
@@ -11,15 +11,38 @@
...
+     * @var string
+     */
+    private $staticContentPath;
+
+    /**
+     * @param DocumentManagerInterface $dm
+     * @param ClassMetadata $class
+     * @param string $staticContentPath
+     */
+    public function __construct(DocumentManagerInterface $dm, ClassMetadata $class, $staticContentPath)
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ContentBundle/Doctrine/ODM/PHPCR/StaticContentRepository.php]: Why did you abbreviate?

Comment (pamil) [inline src/Sylius/Bundle/ContentBundle/Doctrine/ODM/PHPCR/StaticContentRepository.php]: To be consistent with parent implementation of `__construct`

File: src/Sylius/Behat/Resources/config/suites/ui/cmf/static_content.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                - sylius.behat.context.hook.doctrine_orm
+                - sylius.behat.context.hook.phpcr
+
+                - sylius.behat.context.transform.shared_storage
+
+                - sylius.behat.context.setup.channel
+                - sylius.behat.context.setup.static_content
+
+                - sylius.behat.context.ui.shop.static_content
+
```
Comment (lchrusciel) [inline src/Sylius/Behat/Resources/config/suites/ui/cmf/static_content.yml]: redundant

Comment (pamil) [inline src/Sylius/Behat/Resources/config/suites/ui/cmf/static_content.yml]: Srsly? It just separates contexts from filters.

Comment (lchrusciel) [inline src/Sylius/Behat/Resources/config/suites/ui/cmf/static_content.yml]: Yup. Indention separates them enough. We don't have such a separation in any other suite.

File: src/Sylius/Bundle/ShopBundle/Resources/views/StaticContent/show.html.twig [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{% extends '@SyliusShop/layout.html.twig' %}
+
+{% block content %}
+    {% createphp resource as='rdf' noautotag %}
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/ShopBundle/Resources/views/StaticContent/show.html.twig]: We should not use createphp for new shop. Please remove it.

**Comment by pjedrzejewski** [comment]
Thanks Kamil!

**Pull request merged**